README.md

```diff
@@ -20,10 +20,85 @@ Living outside of the core product, Kyma allows you to be completely language-ag
+  ├── azure.Jenkinsfile           # The file that builds and tests the Azure cluster
+  ├── ci.Dockerfile               # The file that verifies and tests the Docker image built
+  ├── governance.Jenkinsfile      # The file that validates external links in the documentation
+  └── orchestrator.Jenkinsfile    # The file that triggers Kyma integration and runs the CI files of all subprojects based on the introduced changes     
+  ```
+
 ## Installation
 
 Install Kyma [locally](docs/kyma/docs/031-gs-local-installation.md) and on a [cluster](docs/kyma/docs/032-gs-cluster-installation.md).
 
 ## Usage
 
 Kyma comes with the ready-to-use code snippets that you can use to test the extensions and the core functionality. See the list of existing examples in the [`examples`](https://github.com/kyma-project/examples) repository.
+
+## Development
+
+Develop on your remote repository forked from the original repository in Go.
+See the example that uses the [`ui-api-layer`](components/ui-api-layer) project located in the `components` directory in the `kyma` repository but applies to any Go project. This set of instructions uses the recommended [`git workflow`](https://github.com/kyma-project/community/blob/master/git-workflow.md).
+
+1. Fork the repository in GitHub.
+2. Clone the fork to your `$GOPATH` workspace. Use this command to create the folder structure and clone the repository under the correct location:
+
+```
+git clone git@github.com:{github-username}/kyma.git $GOPATH/src/github.com/{github-username}/kyma
+```
+3. Add a Git remote for the original repository:
+
+```
+git remote add upstream https://github.com/kyma-project/community.git
+```
+
+4. Run the `git fetch --all` command to fetch all remote branches.
+
+5. Set up the local `master` branch to track the remote master branch from the upstream repository:
+
+```
+git branch -u upstream/master master
+```
+
+6. Install dependencies.
+
+Go to the main directory of the project in your workspace location and install required dependencies:
+
+```
+$ cd $GOPATH/src/github.com/{github-username}/kyma/components/ui-api-layer
+$ dep ensure -vendor-only
+```
+
+7. Build the project.
+
+Every project runs differently. Follow instructions in the main `README.md` document of the given project to build it.
+
+8. Create a branch, develop, commit, push, squash, and rebase following the recommended [`git workflow`](https://github.com/kyma-project/community/blob/master/git-workflow.md).
+
+9. Run unit tests.
+
+After you finish, test your changes. Use the `go test ./...` command to run unit tests or follow test instructions specified in the main `README.md` document of the given project.
+
+10. Perform acceptance tests.
+
+Go to the `tests` directory in the `kyma` repository and run acceptance tests for a given components.
+
+11. When the tests complete successfully, submit the pull request and wait for the owners' review.
```

Comment:
we need to explain that first is a Pr with code changes, once it is accepted, there must be another Pr with version update in Kyma charts. This flow must be explained, that contribution has for now 2 stages